### PR TITLE
[1LP][RFR]added windows domain credential tests

### PR DESCRIPTION
**Pull request by lkhomenk**
Purpose or Intent
=================
__Extending__  zone tests. verifying that we can add/remove Windows domain credentials

### CODE REVIEW

File: cfme/tests/configure/test_zones.py [hunk 1]
```diff
@@ -121,3 +122,51 @@ def test_zone_add_blank_description():
...
+@pytest.mark.sauce
+def test_add_zone_windows_domain_credentials(request):
+    """
+    Testing Windows Domain credentials add
+    """
+    zc = current_appliance.collections.zones.all()
+    values = {'username': 'userid',
+              'password': 'password',
+              'verify': 'password'}
+    zc[0].update(values)
```
Comment (lkhomenk) [inline cfme/tests/configure/test_zones.py]: using default zone as this operation is safe and doesn't need any provider. cleanup is also here.

File: cfme/tests/configure/test_zones.py [hunk 2]
```diff
@@ -121,3 +122,51 @@ def test_zone_add_blank_description():
...
+@pytest.mark.sauce
+def test_remove_zone_windows_domain_credentials():
+    """
+    Testing Windows Domain credentials removal
+    """
+    zc = current_appliance.collections.zones.all()
+    values = {'username': 'userid',
+              'password': 'password',
+              'verify': 'password'}
+    zc[0].update(values)
```
Comment (lkhomenk) [inline cfme/tests/configure/test_zones.py]: no fixtures as this is not needed anywhere. one time usage atm

**Comment by lkhomenk** [comment]
on 5.8 only test_zone_add_dupe  failed. which is not affected by these code changes

File: cfme/tests/configure/test_zones.py [hunk 3]
```diff
@@ -121,3 +122,51 @@ def test_zone_add_blank_description():
...
+    zc_username = view.username.read()
+    assert zc_username == values['username'], "Username wasn't updated".format(zc_username)
+    remove_values = {'username': '',
+                     'password': '',
+                     'verify': ''}
+    zc[0].update(remove_values)
+    view = navigate_to(zc[0], 'Edit')
+    removed_zc_username = view.username.read()
+    assert removed_zc_username == remove_values['username'], "Username wasn't removed".format(
+        zc_username)
```
Comment (quarckster) [inline cfme/tests/configure/test_zones.py]: **REQUIRED**
There is no `{}` in the string :)

File: cfme/tests/configure/test_zones.py [hunk 4]
```diff
@@ -121,3 +122,51 @@ def test_zone_add_blank_description():
...
+    Testing Windows Domain credentials removal
+    """
+    zc = current_appliance.collections.zones.all()
+    values = {'username': 'userid',
+              'password': 'password',
+              'verify': 'password'}
+    zc[0].update(values)
+    view = navigate_to(zc[0], 'Edit')
+    zc_username = view.username.read()
+    assert zc_username == values['username'], "Username wasn't updated".format(zc_username)
```
Comment (quarckster) [inline cfme/tests/configure/test_zones.py]: REQUIRED
There is no {} in the string :)

**Pull request merged**